.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,19 +13,54 @@ For questions, the best place to get answers is on our [discussion forum](https:
 
 Please note: We take Boundary's security and our users' trust very seriously. If you believe you have found a security issue in Boundary, please responsibly disclose by contacting us at security@hashicorp.com. Our PGP key is available at [our security page](https://www.hashicorp.com/security/).
 
+If Boundary Desktop Client version >= 2.1.0
+
+Logs are found at:
+
+Linux: ~/.config/Boundary/logs/desktop-client.log
+MacOS: ~/Library/Logs/Boundary/desktop-client.log
+Windows: $env:USERPROFILE\AppData\Roaming\Boundary\logs\desktop-client.log
+
+If requests are taking longer than expected, please change the logging level to DEBUG. This will log the cache daemon request times as well.
+
 -->
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+## :bug: Bug Report
+
+### **Description**
+
+_A clear and concise description of the bug._
+
+### **Steps to Reproduce**
+
+1. Go to '...'
+2. Click on '...'
+3. Scroll down to '...'
+4. See the error
+
+## **Expected behavior**
+
+_A clear description of what you expected to happen._
+
+## **Actual behavior**
+
+_A clear description of what actually happened._
+
+## **Screenshots (if applicable)**
+
+_Add screenshots or screen recordings to help explain the issue._
+
+## **Environment**
+
+- **OS:** (e.g., Windows 10, macOS 14.0)
```

Comment:
I would also include asking for the architecture and specifically mention the OS version too

Reply:
Sorry, I had that initially and didn't copy it over while refactoring. Thanks for catching that.